api/scripts/create-certification-center-memberships-from-organization-admins.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const _ = require('lodash');
+
+const { checkCsvExtensionFile, parseCsvWithHeader } = require('./helpers/csvHelpers');
+
+const { NotFoundError } = require('../lib/domain/errors');
+const Membership = require('../lib/domain/models/Membership');
+
+const Bookshelf = require('../lib/infrastructure/bookshelf');
+const bookshelfToDomainConverter = require('../lib/infrastructure/utils/bookshelf-to-domain-converter');
+const BookshelfCertificationCenter = require('../lib/infrastructure/data/certification-center');
+const BookshelfMembership = require('../lib/infrastructure/data/membership');
+
+function getCertificationCenterByExternalId(externalId) {
+  return BookshelfCertificationCenter
+    .where({ externalId })
+    .fetch({ require: true })
+    .then((certificationCenter) => bookshelfToDomainConverter.buildDomainObject(BookshelfCertificationCenter, certificationCenter))
+    .catch((err) => {
+      if (err instanceof BookshelfCertificationCenter.NotFoundError) {
+        throw new NotFoundError(`CertificationCenter not found for External ID ${externalId}`);
+      }
+      throw err;
+    });
+}
+
+async function getAdminMembershipsByOrganizationExternalId(externalId) {
+  return BookshelfMembership
+    .query((qb) => {
+      qb.innerJoin('organizations', 'memberships.organizationId', 'organizations.id');
+      qb.where('organizationRole', Membership.roles.ADMIN);
+      qb.where('organizations.externalId', '=' , externalId);
+    })
+    .fetchAll({ required: true })
```

Comment:
Ici c'est `require` comme dans l'appel de la methode `fetch` un peu plus haut ?
 
```suggestion
    .fetchAll({ require: true })
```

Reply:
Exact !